public/app/features/transformers/spatial/SpatialTransformerEditor.tsx

```diff
@@ -165,4 +167,5 @@ export const spatialTransformRegistryItem: TransformerRegistryItem<SpatialTransf
   description: spatialTransformer.description,
   state: PluginState.alpha,
   categories: new Set([TransformerCategory.PerformSpatialOperations]),
+  help: getTransformationContent(DataTransformerID.spatial).helperDocs,
```

Comment:
I noticed the spatial docs were quite sparse. Probably not something for this PR, but we should make a follow up PR for it.

Reply:
Agreed. That's the final piece of this epic: expanding/correcting the docs themselves.